projects/observability/src/shared/components/cartesian/d3/chart/cartesian-chart.ts

```diff
@@ -281,6 +305,20 @@ export class DefaultCartesianChart<TData> implements CartesianChart<TData> {
     this.moveDataOnTopOfAxes();
     this.drawMouseEventContainer();
     this.setupEventListeners();
+    this.attachBrush();
```

Comment:
Should be config driver as mentioned above.

Reply:
also could be part of setupEventListerners

Reply:
done